daphne/src/messages/mod.rs

```diff
@@ -706,36 +754,51 @@ impl Default for Query {
 /// A collect request.
 //
 // TODO Add serialization tests.
+// XXX Update for draft04
 #[derive(Clone, Debug, Deserialize, PartialEq, Eq, Serialize)]
 pub struct CollectReq {
```

Comment:
For consistency with draft04?
```suggestion
pub struct CollectionReq {
```

Reply:
Yeah, we probably should do that.  If we do, we need to do it for the other affected structs:

CollectResp -> Collection
AggregateShareResp -> AggregateShare